lib/routes/api/accounts.js

```diff
@@ -1062,8 +1062,14 @@ const updateCall = async(req, res) => {
       });
 
       if (!response.ok) {
-        logger.error(`Error sending updateCall POST to ${url}`);
```

Comment:
I think we should have kept this line in, as now we have no visibility in the logs for these non 200 responses, things like a 422 to the user make sense but there could be other reasons the feature server returns say a 5xx response that we'd want to see

Reply:
fixed